config/application.rb

```diff
@@ -41,5 +42,10 @@ class Application < Rails::Application
 
     # Path within public/ where assets are compiled to
     config.assets.prefix = '/government-frontend'
+
+    # Do not swallow errors in after_commit/after_rollback callbacks.
+    # config.active_record.raise_in_transactional_callbacks = true
+
+    GdsApi.config.always_raise_for_not_found = true
```

Comment:
On second thoughts, the deprecation warning is telling us we should no longer use `get_json`, `post_json` etc, instead we should use `get_json!`, `post_json!` etc.

Can we update our code so that we don't need to configure this option? If we can easily do it now we won't need to revisit before the 1st October 2016 when the feature is due to be removed.